### PR TITLE
Contributing.md

### PR DESCRIPTION
**Pull request by HarryHeights**
This is the contribution guide I promised to create. I guessed some things because I wasn't so sure. 

Are you okay with the fact that contributors can contribute however they like to the project? or you'll prefer they discuss with you first via an issue or an email.

### CODE REVIEW

File: Contyributing.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers. <hr>
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at activity-diary@rampro.de. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (tadly) [inline Contyributing.md]: `activity-diary@rampro.de` that's not quite right now is it :)
For now you can use `support@linuxwhatelse.de`.

**Review by tadly — CHANGES_REQUESTED** [review]
Just some typos I noticed that need fixing.
Apart from that this is looking good :)

**Review by tadly — APPROVED** [review]

**Comment by tadly** [comment]
I just noticed...
Could it be that the filename is `Contyributing.md`?

**Comment by HarryHeights** [comment]
The filename is a mistake.

**Comment by HarryHeights** [comment]
I think I changed the name in the last commit

**Comment by tadly** [comment]
Not really, if you check your most recent commit a93245ca081cc4f67582da89442c8d2cd239cc56, it's still saying `Contyributing.md`

**Comment by HarryHeights** [comment]
Okay, lemme change it

**Pull request merged**